ros2launch/examples/includes_example.launch.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Include and launch the example.launch.py launch file."""
+
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.actions import LogInfo
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import ThisLaunchFileDir
+
+
+def generate_launch_description():
+    """Launch the example.launch.py launch file."""
+    return LaunchDescription([
+        LogInfo(msg=[
+            'Including launch file located at: ', ThisLaunchFileDir(), '/example.launch.py'
```

Comment:
You should always, because when introspecting it will be the difference `/abs/path/to/example.launch.py` and `ThisLaunchFileDir('/abs/path/to') + '/example.launch.py'`. Which seems trivial, but it indicates that if you moved the launch file it would work, as opposed to it being a hard coded path.